SortApp/SortApp.BL.Test/SortEngineTests.cs

```diff
@@ -26,7 +26,7 @@ public void SortArrayFilledTest()
 			{
 				GenerateFromStringString = (s) => new[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 }
 			};
-
+			
```

Comment:
I'm not quite sure what changed here. Please, look through you changes before you commit them to make sure such things not come to changes. It's has no use and only confuses.